DXMainClient/Domain/Multiplayer/GameOptionPresets.cs

```diff
@@ -127,6 +127,16 @@ public GameOptionPreset GetPreset(string name)
             return null;
         }
 
+        public List<string> GetPresetNames()
+        {
+            if (gameOptionPresetsIni == null)
+                LoadIni();
```

Comment:
This specific check is repeated so many times in the code that at this point I think it could be good to add a function that just had these two lines, and then just call the function everywhere where these two lines currently exist. `LoadIniIfNotInitialized` or something along those lines.

Reply:
Done.